dev/source/docs/style-guide.rst

```diff
@@ -467,3 +467,36 @@ The user field helps to categorize and hide advanced parameters from being adjus
 
 - Standard - Available to anyone
 - Advanced - Available to advanced users
+
+Floating Point Annotation
+=========================
+
+ArduPilot is compiled with ``-fsingle-precision-constant``.
+
+That means it is currently allowable to leave off the float specifier from constants.  It is also permissable to have them.
```

Comment:
I would rather say : please use them ... otherwise when we will actually need double we will have issues !

Reply:
I disagree, it makes the code harder to read